packages/inter-protocol/scripts/build-bundles.js

```diff
@@ -1,36 +1,20 @@
 #! /usr/bin/env node
 import '@endo/init';
-import {
-  createBundles,
-  extractProposalBundles,
-} from '@agoric/internal/src/node/createBundles.js';
-import url from 'url';
-import process from 'process';
-
-import { defaultProposalBuilder } from './init-core.js';
-import { defaultProposalBuilder as collateralProposalBuilder } from './add-collateral-core.js';
 
-process.env.INTERCHAIN_ISSUER_BOARD_ID =
-  'arbitrary value to prevent build error';
+import { createBundles } from '@agoric/internal/src/node/createBundles.js';
+import url from 'url';
 
 const dirname = url.fileURLToPath(new URL('.', import.meta.url));
-await extractProposalBundles(
```

Comment:
that is passing

Reply:
cosmic-swingset failing tho https://github.com/Agoric/agoric-sdk/actions/runs/5826023109/job/15799109656?pr=8150

@michaelfig  can `extractProposalBundles` move somewhere that doesn't create a cycle?

if not, should `builders` have an Inter bundle builder for use in cosmic-swingset?

Reply:
That was failing for a different reason. I added a diagnostic which helped nail it down.

A different module, `measurement-bootstrap.js` was expecting the `bundle-vaultFactory.js` that this removed. Since that tool measure bootstrap it doesn't belong in swingset-vat. I couldn't find anyone who claimed to own it so this PR deletes it for now. If a need comes up we can revert that change and put it into a compatible place.